src/components/Search/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+
+// import {  } from './index.module.css'
+
+class SearchInput extends Component<*, *> {
+  componentDidMount() {
+    // console.log('SearchInput mounted')
+  }
+
+  render() {
+    // const {  } = this.props
+
+    return (
+      <div>
+        SearchInput
```

Comment:
How does this work?

Reply:
This is just a text which will be replaced with some content. @maybelinot